server/src/main/java/org/elasticsearch/action/admin/cluster/node/stats/NodesStatsRequest.java

```diff
@@ -148,9 +149,24 @@ public NodesStatsRequest removeMetric(String metric) {
         return this;
     }
 
+    @Override
+    public String getDescription() {
+        return Strings.format(
+            "nodes=%s, metrics=%s, flags=%s",
```

Comment:
Should this also include at the beginning something like "nodes stats request"?

Reply:
The description is rendered alongside the action name so I don't think that's needed:

```
        "h5WLX6K_T4WWFsErhRroBQ:190": {
          "node": "h5WLX6K_T4WWFsErhRroBQ",
          "id": 190,
          "type": "direct",
          "action": "cluster:monitor/nodes/stats[n]",
          "description": "nodes=[], metrics=[jvm, process, os, breaker, thread_pool, transport, fs, script, ingest, indexing_pressure, repositories, discovery, http, script_cache, adaptive_selection], flags=[Store, Indexing, Get, Search, Merge, Flush, Refresh, QueryCache, FieldData, Docs, Warmer, Completion, Segments, Translog, RequestCache, Recovery, Bulk, Shards, Mappings, DenseVector]",
          "start_time_in_millis": 1693248714558,
          "running_time_in_nanos": 1990622708,
          "cancellable": true,
          "cancelled": true,
          "parent_task_id": "h5WLX6K_T4WWFsErhRroBQ:189",
          "headers": {}
        }
```